### PR TITLE
Revert part of #680 to unblock 4.1/4.2/4.3 CI

### PR DESCRIPTION
**Pull request by liangxia**
https://github.com/openshift/verification-tests/pull/680

### CODE REVIEW

**Comment by liangxia** [comment]
Once the 'oc config' bug is fix, we can switch to --kubeconfig

**Comment by liangxia** [comment]
/hold
Choose one of #702 and #703

**Comment by liangxia** [comment]
/hold
To have more time/people to think about those issues.

File: lib/rules/cli/4.1.yaml [hunk 1]
```diff
@@ -1110,7 +1110,7 @@
   :cmd: oc adm diagnostics
   :options:
     :cluster-context: --cluster-context=<value>
-    :config: --kubeconfig=<value>
+    :config: --config=<value>
```
Comment (akostadinov) [inline lib/rules/cli/4.1.yaml]: this can be removed given it is already in global options

**Pull request closed without merging**